.vscode/settings.json

```diff
@@ -7,13 +7,7 @@
       "source.organizeImports": "explicit"
     }
   },
-  "python.testing.pytestArgs": [
-    ".",
-    "--ignore",
-    "electricitymap/contrib/parsers",
-    "--ignore",
-    "electricitymap/contrib/validators"
```

Comment:
Technically unrelated but the VSCode test runner wouldn't let me run the tests since we no longer have the validators folder.